lib/mustache/generator.rb

```diff
@@ -152,6 +152,34 @@ def on_partial(name, offset, indentation)
       ev("ctx.partial(#{name.to_sym.inspect}, #{indentation.inspect})")
     end
 
+    # Fired when the compiler finds a inheritance. We want to return code
+    # which stores the block variables in this template and  
+    # calls the parent at runtime.
+    def on_parent(name, offset, content, raw, delims)
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Unused method argument - `offset`. If it's necessary, use `_` or `_offset` as an argument name to indicate that it won't be used.](https://www.codacy.com/app/rokusu/mustache/file/1131006119/issues/source?bid=2903613&fileBranchId=2904056#l158)